### PR TITLE
Added italian translations

### PR DESCRIPTION
**Pull request by GioPan04**
Hi,
I added the translations for Italian language.

### CODE REVIEW

File: ubuntu_desktop_installer/lib/l10n/messages_it.dart [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+final messages = new MessageLookup();
+
+typedef String MessageIfAbsent(String messageStr, List<dynamic> args);
+
+class MessageLookup extends MessageLookupByLibrary {
+  String get localeName => 'it';
+
+  static m0(url) => "Per istruzioni, apri questa pagina da un telefono o un altro dispositivo: <a href=\"https://${url}\">${url}</a>";
+
+  static m1(url) => "Forse vorresti leggere le <a href=\"${url}\">informazioni di rilascio</a>.";
```
Comment (oSoMoN) [inline ubuntu_desktop_installer/lib/l10n/messages_it.dart:24]: `m0` and `m1` are missing an explicit return type (`String`), see e.g. messages_fr.dart. This is [a bug in dart itself](https://github.com/dart-lang/intl_translation/issues/99), for now you will have to manually add them.

File: ubuntu_desktop_installer/lib/l10n/intl_it.arb [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  "@Repairing will reinstall all installed software without touching documents or settings.": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Try Ubuntu": "Prova Ubuntu",
+  "@Try Ubuntu": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "You can try Ubuntu without making any changes to your computer.": "Puoi provare Ubuntu senza apportare alcuna modifica al tuo computer",
```
Comment (oSoMoN) [inline ubuntu_desktop_installer/lib/l10n/intl_it.arb:43]: There should probably be a full stop at the end of the sentence, for consistency.

File: ubuntu_desktop_installer/lib/l10n/intl_it.arb [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  "@Try or install": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Repair installation": "Ripara installazione",
+  "@Repair installation": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Repairing will reinstall all installed software without touching documents or settings.": "La riparazione reinstallerá tutti i programmi senza toccare alcun documento o impostazione",
```
Comment (oSoMoN) [inline ubuntu_desktop_installer/lib/l10n/intl_it.arb:33]: There should probably be a full stop at the end of the sentence, for consistency.

**Review by oSoMoN — CHANGES_REQUESTED** [review]
This looks good to me (I can't really speak Italian though, so I'm not reviewing the correctness of the translations), thanks for these!

See a couple of minor suggestions inline.

Note that for testing purposes, I had to add an entry for Italian to `subiquity_client/subiquity/languagelist`.

**Comment by GioPan04** [comment]
Okay done, i push a commit

**Comment by FedericoAntoniazzi** [comment]
I speak italian and translations made by @GioPan04 are perfect. Lgtm

**Comment by oSoMoN** [comment]
Thanks @GioPan04 and @FedericoAntoniazzi, this is now merged.

**Pull request merged**